docs/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - New `EVM.NodePool.SelectionMode` `TotalDifficulty` to use the node with the greatest total difficulty.
+- `EVM.NodePool.SyncThreshold` to ensure that live nodes do not lag too far behind.
```

Comment:
We may wish to add a bit more details here. Perhaps copy/paste from the docs?

Reply:
Not sure what the best format is, quoting sort of works, but then code is also nested :shrug:

Reply:
I think this is fine